.github/workflows/ci.yml

```
@@ -26,7 +26,7 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           cache: 'pnpm'
-          node-version: 18.x
+          node-version: 22.x
```

since you have the .nvmrc file, this can become very conveniently:


```suggestion
          node-version-file: .nvmrc
```